scripts/distances/align.py

```diff
@@ -0,0 +1,663 @@
+from collections import Counter
+import math
+import distance
+from munkres import Munkres, print_matrix
+import numpy as np
+from textutil import break2sentences, extract_terminals
+import evaluation
+import re
+import string
+import zss
+from ucca import layer0, layer1
+
+import inspect
+
+def compare(n1, n2):
+	return True if labels(n1) & labels(n2) else False
```

Comment:
This is just like returning `labels(n1) & labels(n2)`, isn't it? Or maybe `bool(labels(n1) & labels(n2))`.